src/AcceptanceTests/Infrastructure/DefaultServer.cs

```diff
@@ -38,6 +38,7 @@ public Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor,
             var recoverability = configuration.Recoverability();
             recoverability.Delayed(delayed => delayed.NumberOfRetries(0));
             recoverability.Immediate(immediate => immediate.NumberOfRetries(0));
+            configuration.SendFailedMessagesTo("error");
 
             var transportConfig = configuration.UseTransport<MsmqTransport>();
```

Comment:
I'll be more than happy convert these tests to use the learning transport.

Very likely it is not the best place to have this discussion, though.